src/Notifynder/Traits/Notifable.php

```diff
@@ -49,11 +49,11 @@ public function notifications()
     }
 
     /**
-     * Get the notifications Relationship.
+     * Get the notifications Relationship without any eager loading.
      *
      * @return \Illuminate\Database\Eloquent\Relations\HasMany|\Illuminate\Database\Eloquent\Relations\MorphMany
      */
-    public function getNotificationRelation()
+    private function getLazyLoadedNotificationRelation()
```

Comment:
I added the getLazyLoadedNotification method to avoid code duplication of the getNotificationRelation logic. Otherwise we would need to write the same logic in Notifable and NotifableL53